docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-bootstrap-switches-ref.adoc

```diff
@@ -0,0 +1,246 @@
+
+|`org.kie.server.controller`
+|Comma-separated list
+|N/A
+|A comma-separated list of URLs to the controller REST endpoints, for example `http://localhost:8080/business-central/rest/controller`. Setting this property is required when using a controller.
+
+|`org.kie.server.controller.user`
+|String
+|`kieserver`
+|The user name to connect to the controller REST API. Setting this property is required when using a controller.
+
+|`org.kie.server.controller.pwd`
+|String
+|`kieserver1!`
+|The password to connect to the controller REST API. Setting this property is required when using a controller.
+
+|`org.kie.server.controller.token`
+|String
+|N/A
+|This property enables you to use a token-based authentication between the {KIE_SERVER} and the controller instead of the basic user name/password authentication. The server sends the token as a parameter in the request header. Note that long-lived access tokens are required as the tokens are not refreshed.
+
+|`org.kie.server.controller.connect`
+|Long
+|`10000`
+|The waiting time in milliseconds between repeated attempts to connect the {KIE_SERVER} to the controller when the server starts.
+|===
+
+ifdef::BA[]
+.Bootstrap Switches for Persistence Properties
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.server.persistence.ds`
+|String
+|N/A
+|A data source JNDI name. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.tm`
+|String
+|N/A
+|A transaction manager platform for Hibernate properties set. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.dialect`
+|String
+|N/A
+|The Hibernate dialect to be used. Set this property when enabling the BPM support.
+
+|`org.kie.server.persistence.schema`
+|String
+|N/A
+|The database schema to be used.
+|===
+endif::BA[]
+
+.Bootstrap Switches for Executor Properties
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.executor.interval`
+|Integer
+|`3`
+|The time between the moment the {PRODUCT} executor finishes a job and the moment it starts a new one, in a time unit specified in the `org.kie.executor.timeunit` property.
+
+|`org.kie.executor.timeunit`
+| https://docs.oracle.com/javase/8/docs/api/index.html?java/util/concurrent/TimeUnit.html[`java.util.concurrent.TimeUnit`] constant
+|`SECONDS`
+|The time unit in which the `org.kie.executor.interval` property is specified.
+
+|`org.kie.executor.pool.size`
+|Integer
+|`1`
+|The number of threads used by the {PRODUCT} executor.
+
+|`org.kie.executor.retry.count`
+|Integer
+|`3`
+|The number of retries the {PRODUCT} executor attempts on a failed job.
+|===
+
+ifdef::BA[]
+.Callback Bootstrap Switches
+[cols="2,1a,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.jbpm.ht.callback`
+|`mvel`
+
+`ldap`
+
+`db`
+
+`jaas`
+
+`props`
+
+`custom`
+|`jaas`
+|Specifies the implementation of user group callback to be used:
```

Comment:
Can we say:
"A property that specifies the implementation of ...."

Reply:
fixed